src/client-reducer.js

```diff
@@ -1,8 +1,12 @@
 import * as actions from "./game/actions";
+import * as clientActions from "./client-actions";
 
 const INITIAL_STATE = {
   sync: true,
-  desyncStates: {}
+  desyncStates: {},
+  targettingUnit: null,
```

Comment:
i have bad news, and also made the same mistake... `targeting` is spelled with one t 😢